dts/arm/nordic/nrf54l15_cpuapp_peripherals.dtsi

```diff
@@ -261,6 +261,30 @@ timer24: timer@ce000 {
 	prescaler = <0>;
 };
 
+pwm20: pwm@d2000 {
+	compatible = "nordic,nrf-pwm";
+	status = "disabled";
+	reg = <0xd2000 0x1000>;
```

Comment:
Please add also corresponding entries in `validate_base_addresses.c`.

Reply:
done